js/foundation.drilldown.js

```diff
@@ -85,7 +85,11 @@ class Drilldown {
       var $menu = $(this),
           $back = $menu.find('.js-drilldown-back');
       if(!$back.length){
-        $menu.prepend(_this.options.backButton);
+        if(_this.options.backButtonPosition == 'bottom'){
+          $menu.append(_this.options.backButton);
+        } else {
+          $menu.prepend(_this.options.backButton);
+        }
```

Comment:
I don't think this is a good idea to consider all values different from "bottom" as "top". This hide a bug if `bottom` is incorrectly written. We should check for `bottom` and `top` with `switch...case`, and throw an error on an unhandled value.

(poke @kball, what do you think ?)